src/docs/output-targets/custom-elements.md

```diff
@@ -37,6 +37,19 @@ import { HelloWorld } from 'my-library/dist/components/hello-world';
 customElements.define('hello-world', HelloWorld);
 ```
 
+The output directory will also contain an `index.js` file which imports and
+re-exports all of your components and their respective `defineCustomElement`
+helper functions, looking something like this:
+
+```tsx
+export { setAssetPath, setPlatformOptions } from '@stencil/core/internal/client';
+export { MyComponent, defineCustomElement as defineCustomElementMyComponent } from './my-component.js';
+export {
+  MyOtherComponent,
+  defineCustomElement as defineCustomElementMyOtherComponent
+} from './my-other-component.js';
+```
+
```

Comment:
Can we add a snippet of how this could be used as well?  Right now, we tell folks that we'll export this file for them, but we don't give them much instruction as to how to use it compared to the first option (and probably should make that callout that this is an either/or proposition)

Reply:
that makes sense, I'll add an explanation of the specific use-case of setting this ad `module` in `package.json`, do you think there's anything else we should call out?

Reply:
just realized there's a bit of a wrinkle with this...